### PR TITLE
Latex parts list. Issue #20

### PR DESCRIPTION
**Pull request by xtream1101**

### CODE REVIEW

File: Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex [hunk 1]
```diff
@@ -53,10 +66,20 @@
...
+    \centering
+    \arrayrulecolor{lightgray}
+    \sffamily\footnotesize
+    \captionsetup{font={sf,bf}}
+    \caption{Parts/Tools Necessary}
+    \begin{tabular}{|N|Q|Q|I|N|Q|Q|I|}
+        \hline
+        \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} & \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} \\
+        \hline
+        3" Channel & S2 & 4 & \partimg{../../../images/parts_list/S2.jpg} & Metal Hacksaw or Bandsaw & & & \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex]: Include photo of hacksaw and bandsaw

File: Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex [hunk 2]
```diff
@@ -90,16 +113,30 @@ \section{Mechanical/Structural Assembly}
...
+    \centering
+    \arrayrulecolor{lightgray}
+    \sffamily\footnotesize
+    \captionsetup{font={sf,bf}}
+    \caption{Parts/Tools Necessary}
+    \begin{tabular}{|N|Q|Q|I|N|Q|Q|I|}
+        \hline
+        \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} & \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} \\
+        \hline
+        3" Channel (Modified) & S2A & 4 & \partimg{../../../images/parts_list/S2.jpg} & \#6 Washer & W1 & 28 & \partimg{../../../images/parts_list/W1.png} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex]: Include exact photo / CAD of modified part

File: Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex [hunk 3]
```diff
@@ -90,16 +113,30 @@ \section{Mechanical/Structural Assembly}
...
+        3.75" Channel & S3 & 4 & \partimg{../../../images/parts_list/S3.jpg} & 0.5" Nylon Washer & W3 & 12 & \partimg{../../../images/parts_list/W3.png} \\ \hline
+        9" Channel & S5 & 2 & \partimg{../../../images/parts_list/S5.jpg} & \#6 x 1/4 Standoff & T1 & 32 & \partimg{../../../images/parts_list/T1.png} \\ \hline
+        Pattern F Bracket & S7 & 8 & \partimg{../../../images/parts_list/S7.jpg} & \#6-32 x 3/8" Button Head Screw & B2 & 52 & \partimg{../../../images/parts_list/B2.png} \\ \hline
+        0.5” Bore Pillow Block & S11 & 4 & \partimg{../../../images/parts_list/S11.jpg} & \#6-32 x 1/2" Button Head Screw & B3 & 32 & \partimg{../../../images/parts_list/B3.png} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex]: Please reference this part as 0.5" Pillow Block (remove 'Bore')

File: Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex [hunk 4]
```diff
@@ -90,16 +113,30 @@ \section{Mechanical/Structural Assembly}
...
+        0.5” Face Tapped Clamping Hub & S13 & 2 & \partimg{../../../images/parts_list/S13.jpg} & \#6-32 Locking Hex Net & B11 & 60 & \partimg{../../../images/parts_list/B11.png} \\ \hline
+        0.5" x 3" Aluminum Tube & S18A & 2 & \partimg{../../../images/parts_list/S18.jpg} & Allen Key Set & & & \partimg{../../../images/parts_list/D2.jpeg} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Rocker-Bogie/Latex Docs/Rocker-Bogie Build Doc.tex]: Include exact photo of modified part S18A

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 5]
```diff
@@ -57,10 +70,20 @@ \section{Machining/Fabrication}
...
+    \centering
+    \arrayrulecolor{lightgray}
+    \sffamily\footnotesize
+    \captionsetup{font={sf,bf}}
+    \caption{Parts/Tools Necessary}
+    \begin{tabular}{|N|Q|Q|I|N|Q|Q|I|}
+        \hline
+        \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} & \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} \\
+        \hline
+        Wheels & S30 & 6 & \partimg{../../../images/parts_list/S30.jpg} & Center Punch or Starter drill bit & & & \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Include photo of center punch and starter drill bit

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 6]
```diff
@@ -57,10 +70,20 @@ \section{Machining/Fabrication}
...
+        Hand Drill or Drill Press & & & & Drill bit \#23 & & & \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Include photo of hand drill, drill press, and drill bit

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 7]
```diff
@@ -79,20 +102,26 @@ \subsection{Wheel Drilling}
...
+    \centering
+    \arrayrulecolor{lightgray}
+    \sffamily\footnotesize
+    \captionsetup{font={sf,bf}}
+    \caption{Parts/Tools Necessary}
+    \begin{tabular}{|N|Q|Q|I|N|Q|Q|I|}
+        \hline
+        \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} & \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} \\
+        \hline
+        4mm Face Tapped Clamp Hub & S14 & 4 & \partimg{../../../images/parts_list/S14.jpg} & Metal Hacksaw or Bandsaw & & & \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Include pictures of hacksaw and bandsaw

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 8]
```diff
@@ -79,20 +102,26 @@ \subsection{Wheel Drilling}
...
+        Vice or C Clamps & & & & & & & \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Include pictures of vice clamp and c clamp

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 9]
```diff
@@ -109,15 +138,29 @@ \subsection{Cutting the Clamping Hub}
...
+    \arrayrulecolor{lightgray}
+    \sffamily\footnotesize
+    \captionsetup{font={sf,bf}}
+    \caption{Parts/Tools Necessary}
+    \begin{tabular}{|N|Q|Q|I|N|Q|Q|I|}
+        \hline
+        \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} & \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} \\
+        \hline
+        3" Channel & S2 & 6 & \partimg{../../../images/parts_list/S2.jpg} & 172:1 Gear Motor w/Encoder (Drive Motor) & E5 & 6 & \partimg{../../../images/parts_list/E5.jpg} \\ \hline
+        4mm Face Tapped Clamp Hub & S14A & 6 & \partimg{../../../images/parts_list/S14.jpg} & \#6-32 x 1/4" Button Head Screw & B1 & 24 & \partimg{../../../images/parts_list/B1.png} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Include exact photo of modified part S14A

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 10]
```diff
@@ -109,15 +138,29 @@ \subsection{Cutting the Clamping Hub}
...
+        25mm Bore Side Tapped Clamping Mount & S25 & 6 & \partimg{../../../images/parts_list/S25.jpg} & \#6-32 x 1.25" Button Head Screw & B7 & 12 & \partimg{../../../images/parts_list/B7.png} \\ \hline
+        Wheel & S30A & 6 & \partimg{../../../images/parts_list/S30.jpg} & Allen Key Set & & & \partimg{../../../images/parts_list/D2.jpeg} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Wheel should read Wheels.

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 11]
```diff
@@ -109,15 +138,29 @@ \subsection{Cutting the Clamping Hub}
...
+        Loctite & S34 & & \partimg{../../../images/parts_list/S34.jpg} & 5/16 Wrench & & & \partimg{../../../images/parts_list/D1.jpg} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Include photo of Loctite 2-part epoxy, not Loctite threadlocker red

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 12]
```diff
@@ -149,17 +192,31 @@ \subsection{Base Wheel}
...
+    \captionsetup{font={sf,bf}}
+    \caption{Parts/Tools Necessary}
+    \begin{tabular}{|N|Q|Q|I|N|Q|Q|I|}
+        \hline
+        \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} & \thead{Item} & \thead{Ref} & \thead{Qty} & \thead{Image} \\
+        \hline
+        3" Channel & S2 & 4 & \partimg{../../../images/parts_list/S2.jpg} & \#6-32 x 1/4" Screw & B1 & 76 & \partimg{../../../images/parts_list/B1.png} \\ \hline
+        4.5" Channel & S4 & 4 & \partimg{../../../images/parts_list/S4.jpg} & \#6-32 x 1/2" Button Head Screw & B3 & 8 & \partimg{../../../images/parts_list/B3.png} \\ \hline
+        Channel Connector Plate & S6 & 16 & \partimg{../../../images/parts_list/S6.jpg} & \#6-32 x 3/8" Threaded Standoff & T3 & 4 & \partimg{../../../images/parts_list/T3.jpg} \\ \hline
+        Channel Connector Plate & S12 & 4 & \partimg{../../../images/parts_list/S12.jpg} & Allen Key Set & & & \partimg{../../../images/parts_list/D2.jpeg} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: "Channel Connector Plate" should be "0.25" Circular Clamping Hub"

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 13]
```diff
@@ -149,17 +192,31 @@ \subsection{Base Wheel}
...
+        Plain Bore Gear & S26 & 4 & \partimg{../../../images/parts_list/S26.jpg} & 5/16 Wrench & & & \partimg{../../../images/parts_list/D1.jpg} \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: "Plain Bore Gear" should be "48 Tooth Plain Bore Gear"

File: Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex [hunk 14]
```diff
@@ -149,17 +192,31 @@ \subsection{Base Wheel}
...
+        Loctite & S34 & & \partimg{../../../images/parts_list/S34.jpg} & & & & \\ \hline
```
Comment (vssystemluba) [inline Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Name should be 'Loctite Threadlocker', not 'Loctite'.  Ref should be S36, not S34.

Comment (xtream1101) [reply Mechanical/Wheel Assembly/Latex Docs/Wheel Assembly Build Doc.tex]: Same issue as the other. This is S34 in the master parts list and will update the instructions to match.

**Comment by xtream1101** [comment]
@ericjunkins To finish this up I just need a few of the original part images used to create the parts lists.

These were the ones I did not see images for in the repo:
- Vice Clamp or C clamps
- Vice Clamp or V clamps
- Hacksaw or Bandsaw
Body Build Doc
    - Need images for S37A & S37B
Differential Pivot Build Doc
    - Need images of S16B & S18B
Mechanical Intergration Build Doc
    - Need image for S16A
Rocker Bogie Build Doc
    - Need image for S18A
Wheel Assembly
    - Need image for S14A

**Comment by ericjunkins** [comment]
Sorry I did this a little bit complicated I realize to get you the images. There is a branch on the main repo that is "latex-parts-lists" that has a folder with the images that lives at "images/supplemental/" 

https://github.com/nasa-jpl/open-source-rover/tree/latex-parts-list/images/supplemental%20images

this has the images you need. Sorry about the complication here, I was not sure how to add the images best to you to get. (If you have a solution that was easier feel free to let me know!)

**Comment by xtream1101** [comment]
Thanks @ericjunkins that worked out, you can remove that branch now.
@vssystemluba It should be good to go now.

**Comment by vssystemluba** [comment]
Merged to master after some minor styling updates and typo fixes.  This closes issue #20.  Thanks for your help on this!

**Pull request merged**